src/is-authenticated-with-context.js

```diff
@@ -0,0 +1,10 @@
+import Cookies from 'js-cookie'
+
+export default function () {
```

Comment:
Ah, okay. I was thinking of handling getting the specific context within the app, but that probably makes more sense...